internal/tools/securitypolicy/main.go

```diff
@@ -172,8 +184,61 @@ func createPolicyFromConfig(config Config) (sp.SecurityPolicy, error) {
 			container.Layers = append(container.Layers, hashString)
 		}
 
+		// add rules for all known environment variables from the configuration
+		// these are in addition to "other rules" from the policy definition file
+		config, err := img.ConfigFile()
+		if err != nil {
+			return p, err
+		}
+		for _, env := range config.Config.Env {
+			rule := sp.SecurityPolicyEnvironmentVariableRule{
+				Strategy: "string",
+				Rule:     env,
+			}
+
+			container.EnvRules = append(container.EnvRules, rule)
+		}
+
+		// cri adds TERM=xterm for all workload containers. we add to all containers
+		// to prevent any possble erroring
+		rule := sp.SecurityPolicyEnvironmentVariableRule{
+			Strategy: "string",
```

Comment:
Can we have the strategies that we support as consts please. I'm gathering it's just string and re2 at the moment?

Reply:
Oh it's stated in the pr description which schemes are supported :P Ignore my ignorance.

Reply:
Issue added to address in the 2nd half of September.